### ISSUE TITLE
tools/ocpn-metadata generate   verbose has been eliminated.

### ISSUE THREAD

**Issue by rgleason** [OpenCPN/plugins#88]
Sorry, but you will not find me contributing until this is fixed again!

This is the output,  I need to see the printf output.  It can be the failed items or not, but there must be some output to determine what is wrong.  Same would apply to the many many dots that represent URL's.  Otherwise I can't fix anything and I won't be able to add squiddio or vdr which are ready.  Failure to address this problem will result in my withdrawal from this process. Sorry.. 

```
C:\Users\Rick\Documents\GitHub\plugins>python tools/ocpn-metadata generate --userdir metadata --destfile ocpn-plugins.xml
Traceback (most recent call last):
  File "tools/ocpn-metadata", line 137, in <module>
    main()
  File "tools/ocpn-metadata", line 132, in main
    generate(args.userdir, args.destfile, args.version)
  File "tools/ocpn-metadata", line 90, in generate
    subtree = ET.parse(str(path))
  File "C:\Users\Rick\AppData\Local\Programs\Python\Python38-32\lib\xml\etree\ElementTree.py", line 1202, in parse
    tree.parse(source, parser)
  File "C:\Users\Rick\AppData\Local\Programs\Python\Python38-32\lib\xml\etree\ElementTree.py", line 595, in parse
    self._root = parser._parse_whole(source)
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
```

**Comment by bdbcat** [comment]
Rick...
Really, all you have to do is ask.  The drama is not necessary.
1.  I have fixed it in master, (where it has never been before), or...
2.  You might have found it yourself in branch Beta, where it has always been....

**Comment by rgleason** [comment]
I thought it would be everywhere. I ended up starting slowly, back at rg-alpha where things work, got squiddio and vdr working and a good ocpn_plugin.xml.

I then took all metadata in master back to rg-alpha and made a full ocpn_plugins.xml for the master branch.  Then moved that back to master and made a pr.  

There is probably some stuff in there you dont want, that I need in order to easily fix stuff. I can try to remove it in the morning.
